changelog/1078.improvement.md

```diff
@@ -0,0 +1 @@
+Add a `stack` property to the `Tracker` class which corresponds to the dialogue stack.
```

Comment:
Did we clarify with CSE if they need access to the whole stack, or if they need only the top frame which represents the current `context`, in which case the implementation changes slightly?

Reply:
I've asked for clarification.

Reply:
I've clarified that Implementation matches requirement. cc @souvikg10